frontend/init_methods.cpp

```diff
@@ -118,6 +123,7 @@ void init_executors(void) {
     std::cout << "Initialize executors and masks..." << std::endl;
 #ifdef OCTOTIGER_HAVE_KOKKOS
     if (!Kokkos::is_initialized()) { // gets initialized earlier on root locality
+      // TODO SYCL Need args for distributed build...
```

Comment:
Does that means, we can only run single nodes SYCL?

Reply:
For SYCL I often had to set the device id at startup using the CLI arguments. However, as far as I know that can also be done using environment variables, or even by just limiting the number of devices per rank in distributed builds. Hence it shouldn't really be a problem for distributed builds.